Create CONTRIBUTING.MD (#11)
GoogleCloudPlatform/click-to-deploy-solutions: CONTRIBUTING.MD

   build.
2. Update the README.md with details of changes to the interface, this includes new environment 
   variables, exposed ports, useful file locations and container parameters.
3. Increase the version numbers in any examples files and the README.md to the new version that this
   Pull Request would represent. The versioning scheme we use is [SemVer](http://semver.org/).
4. You may merge the Pull Request in once you have the sign-off of two other developers, or if you 
   do not have permission to do that, you may request the second reviewer to merge it for you.

## Project structure
 * Our projects use Terraform to write infrastructure as code, and Cloud Build to run the deploy and destroy pipelines. So your project must have at least the following structure to be deployable:
 * terraform: folder with .tf files
 * terraform.tfvars: with variables values
 * cloudbuild.yaml: deploy pipeline
 * cloudbuild_destroy.yaml: destroy pipeline
 * readme.md: solution documentation
 * architecture.png: architecture diagram that demonstrates the solution
 * cloudbuild.sh: execute pre-reqs and trigger Cloud Build job

## Label Strategy

> ```
> Must have a variable to define resource labels, and aggregate this project's labels with locals.
> locals{
>   resource_labels = merge(var.resource_labels, {
>     deployed_by = "cloudbuild"
>     env         = "sandbox"
>     repo        = "click-to-deploy-solutions"
>     solution    = "private-cloud-data-fusion"
>     terraform   = "true"
>   })
> }
> 
> variable "resource_labels" {
>   type        = map(string)
>   description = "Resource labels"
>   default     = {}
> }
> ```

Reviewer: ```suggestion
Must have a variable to define resource labels, and aggregate this project's labels with locals.

```hcl
locals {
  resource_labels = merge(var.resource_labels, {
    deployed_by = "cloudbuild"
    env         = "sandbox"
    repo        = "click-to-deploy-solutions"
    solution    = "private-cloud-data-fusion"
    terraform   = "true"
  })
}

variable "resource_labels" {
  type        = map(string)
  description = "Resource labels"
  default     = {}
}
```
```

## Cloud SQL

> Since Cloud SQL does not allow you to recreate an instance immediately after deletion due to name conflict, use a random string at the end of the instance name, for example:

Reviewer: ```suggestion
Since Cloud SQL does not allow you to recreate an instance immediately after deletion due to name conflict, add a random suffix to the instance name, for example:
```

Reviewer: Missing the example code here
